.github/workflows/bun-test.yml

```
@@ -0,0 +1,24 @@
+name: Bun Test
+
+on:
+  pull_request: ["main"]
```

```suggestion
  pull_request:
```

We want it to run for all